capstone-app/src/app/modules/onboarding/onboarding.module.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatButtonModule } from '@angular/material/button';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { MatSelectModule } from '@angular/material/select';
+import { ContentComponent } from './content/content.component';
+import { MaterialModule } from '../material/material.module';
+
+@NgModule({
+  declarations: [ContentComponent],
+  imports: [
+    CommonModule,
+    MatButtonModule,
+    FormsModule,
+    ReactiveFormsModule,
+    MatSelectModule,
```

Comment:
You're importing the Material Module so I do not believe the angular material ones are necessary to import here.